qcfractal/cli/qcabalsam_interface.py

```diff
@@ -0,0 +1,44 @@
+"""
+A special CLI function which Balsam can point to and assign as an App to behave in a similar way to other Queue
+Manager's clients.
+
+Users are not expected to manually run this CLI argument and should only be run from Balsam itself as an App
+"""
+
+import os
+import traceback
+
+from ..fractal_utils import get_function
+from qcelemental.models import FailedOperation
+
+
+def main():
```

Comment:
Could we write this once to `tmp` during Balsam startup rather than having it as an actual CLI?

Reply:
I've thought about that, but the path the absolute binary needs to be consistent in order to register it as a Balsam App. Unless we want to re-register a new App for every single job or manager instance.

So my fix for doing that was to have an absolute path from the get go which will be constant between most invocations (at least the same python environment). I thought maybe I could have the user or some configuration point to a file which would be written, but then there is the same App registration problem. As such, my fix was to tie it to the install of QCFractal itself, then the user never has to interact with `ApplicationDescription` object, `BalsamJob` object, or the CLI for balsam